### PR TITLE
Add container health details to status

### PR DESCRIPTION
**Pull request by JazzFisch**

### CODE REVIEW

File: src/components/services/status.jsx [hunk 1]
```diff
@@ -13,6 +13,22 @@ export default function Status({ service }) {
...
+
+    if (data.health === "unhealthy") {
+      return (
+        <div className="w-auto px-1.5 py-0.5 text-center bg-theme-500/10 dark:bg-theme-900/50 rounded-b-[3px] overflow-hidden" title={data.health}>
+          <div className="text-[8px] font-bold text-orange-400/50 dark:text-orange-400/80 uppercase">{data.health}</div>
+        </div>
+      );
+    }
+
     return (
```
Comment (shamoon) [inline src/components/services/status.jsx:32]: Do you want an `else` here? e.g. only show starting / unhealthy / running, I feel like that would communicate the same info

**Review by shamoon — APPROVED** [review]
Nice! Works well for me:
Screen Shot 2022-11-27 at 2 43 39 PM
Screen Shot 2022-11-27 at 2 39 42 PM

**Comment by JazzFisch** [comment]
@shamoon does this most recent change show "healthy" correctly instead of "running" when the container has a healthcheck?

**Comment by shamoon** [comment]
Yep!

Screen Shot 2022-11-27 at 2 57 46 PM

**Pull request merged**